pylint/checkers/exceptions.py

```diff
@@ -152,6 +159,11 @@ def _is_raising(body: typing.List) -> bool:
         "immediately. Remove the raise operator or the entire "
         "try-except-raise block!",
     ),
+    "W0707": (
+        "Exception needs to be raised with cause",
```

Comment:
What would you think of this ?
```suggestion
        "Consider explicitly re-raising using the 'from' keyword", 
```